wiki/en/en-Sound-Devices.md

```diff
@@ -1,87 +1,138 @@
 ---
 layout: wiki
-title: "Known Good Devices"
+title: "Sound Devices"
 lang: "en"
 permalink: "/wiki/Sound-Devices"
 ---
 
-# Known Good Devices
+# Sound Devices
 
-This page lists hardware known to work or not work with Jamulus (although it may vary between Windows, Mac and Linux). Please update with any new information you have (and indicate the platform(s) you have used).
+This page lists (thanks to Jamulus users) the audio devices known to work (or not) with Jamulus.  
+_**Note**: depends on how the device works on your system (Linux, MacOS, Windows, etc.)_.
 
-### Known to Work with Jamulus
+You can update it with the information you have. **Please indicate the platform(s) you are using!**.
```

Comment:
Are you sure that we want this here? Wouldn't it be better in the upcoming knowledge base? But it would spread information.

Furthermore the user wouldn't know how to add anything

Reply:
I don't know what si the best to have information from the users. Comunity page? Issue? Forum?

Reply:
I'd suggest to get the information via the forum (but we could also use an issue here)

Reply:
OK, we link to the forum?
Or we remove this sentence?

Reply:
I'd suggest to add a link to the forum. But let's wait for some other comments.